src/app/modules/submission/email-submission/email-submission.controller.ts

```diff
@@ -374,6 +374,7 @@ const submitEmailModeForm: ControllerHandler<
               // Return the reply early to the submitter
               message: 'Form submission successful.',
               submissionId: submission.id,
+              timestamp: (submission.created || new Date()).getTime(),
```

Comment:
I wonder if `|| new Date()` is overly defensive? Is there really a case where we have a `submission` without a `created` field on it?

(Same question/comment for the encrypt controller below)

Reply:
haha `submission.created` is a `number | undefined` so typescript won't let me set `timestamp` without a default 😅

Reply:
Ah, thanks for the clarification.... Oh well 🙄 🤷